File: Roadmap/00 - SINTAXIS, VARIABLES, TIPOS DE DATOS Y HOLA MUNDO/php/Abel-ADE.php
<?php

// Página Web Oficial: https://www.php.net/manual/es/index.php

//Tipos de comentarios en PHP:

    // Comentario de una sola línea
    # Otro Comentario de una línea
    /*  Comentario 
        multilínea 
    */

    /**
     * Comentario multilínea 
     * que se utiliza 
     * para generar documentación del programa
     */

//Declaración y asignación de variables
$variableCamelCase = 'Hola, Mundo!';

//Declaración y asignación de constantes
const MI_CONSTANTE = 12; //Se define en tiempo de compilación
define('OTRA_CONSTANTE', 50); //Se define en tiempo de ejecución

//Mostrar el contenido
print $variableCamelCase;
print MI_CONSTANTE;
print OTRA_CONSTANTE;

//Tipos de datos primitivos en PHP
$miEntero = 100;
$miDecimal = 20.50;
$miBooleano = true;
$miTexto = 'Soy un texto';
$miNulo = null;

//Imprime por terminal el texto: 'Hola, PHP!'
print 'Hola, PHP!';

File: Roadmap/00 - SINTAXIS, VARIABLES, TIPOS DE DATOS Y HOLA MUNDO/java/AbelADE.java
/**
 * Solución del reto #00 SINTAXIS, VARIABLES, TIPOS DE DATOS Y HOLA MUNDO;
 * @author AbelADE
 */
public class AbelADE {

    /**
     * @param args the command line arguments
     */
    public static void main(String[] args) {
        //Comentario con URL del sitio web oficial = https://docs.oracle.com/javase/8/docs/api/

        // Comentario de simple línea
        
        /*Comentario 
                de varias líneas*/
        
        /**
         * Comentario JavaDoc, para generar documentación oficial
         */
        
        // Crea una variable (y una constante si el lenguaje lo soporta).
        final String TEXTO = "Soy una constante";
        String textoVariable = "Soy una variable";
        
        
        //Crea variables representando todos los tipos de datos primitivos
        boolean datoLogico = true; // Almacena valores true o false
        char caracter = 'a'; // Caracteres unicode de 2 bytes
        byte numeroMuyPequeño = 100; // Entero entre -128 e 127
        short numeroPequeño = 25000; //  Entero entre -32768 y 32767
        int numero = 1000000; // Entero entre -2.147.483.648 y 2.147.483.647
        long numeroMasGrande = 2000000000L; // Entero entre -9.223.372.036.854.775.808 e 9.223.372.036.854.775.807
        float decimalPequeño = 15.6666F; // Decimal de 4 bytes
        double decimal = 15.269899; // Decimal de 8 bytes

        //Tipo de dato no primitivo (Objeto)
        String texto = "Soy un texto";
        
        //Imprime por terminal el texto: "¡Hola, [y el nombre de tu lenguaje]!"
        System.out.println("¡Hola, Java!");
    }

}
